src/index.js

```diff
@@ -2,8 +2,12 @@
 
 const WritableStream = require("./writable_stream");
 
-module.exports = renderer => {
+module.exports = rendererPath => {
 	return (req, res, next) => {
+		const renderer = typeof rendererPath === "function" ?
+			rendererPath	:
```

Comment:
:rofl: sorry, my bad twin was at work, I guess. But funny, that eslint isn't even mentioning it